poetry.lock

```diff
@@ -1,4 +1,4 @@
-# This file is automatically @generated by Poetry 1.7.1 and should not be changed by hand.
```

Comment:
Hmm. This is really silly.

Reply:
This doesn't really matter, I ran poetry (1.7.1) locally, and it has generated the same file (except this comment), so that's fine.

Reply:
Yeah just leads to stupid churn.